### PR TITLE
Add metrics page wb

### PR DESCRIPTION
**Pull request by bakhtw1**
https://user-images.githubusercontent.com/16611277/156959353-b337a22c-b927-4846-8066-1a3ba599e23b.mp4

### CODE REVIEW

File: lib/Screens/metrics.dart [hunk 1]
```diff
@@ -8,16 +14,24 @@ class Metrics extends StatefulWidget {
...
-        ),
-      );
+    return Scaffold(
+      body: ListView(
+        children: <Widget>[
+          // LineChartCard(),
+          // SizedBox(
+          //   height: 20,
+          // ),
+          // LineChartCard(),
```
Comment (AbeerAdeel) [inline lib/Screens/metrics.dart]: Accidentally left commented code here :)

File: lib/components/BarChart.dart [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+        weekData[i] = Random().nextInt(widget.yExtents.toInt()).toDouble() + 6;
+      }
+      refreshState();
+    });
+  }
+
+  Widget DatePicker() {
+    return Container(
+      // width: 250,
+      // height: 450,
```
Comment (AbeerAdeel) [inline lib/components/BarChart.dart]: Here as well

File: lib/components/DoubleBarChart.dart [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+        weekData2[i] = Random().nextInt(widget.yExtents.toInt()).toDouble() + 6;
+      }
+      refreshState();
+    });
+  }
+
+  Widget DatePicker() {
+    return Container(
+      // width: 250,
+      // height: 450,
```
Comment (AbeerAdeel) [inline lib/components/DoubleBarChart.dart]: Here as well

**Comment by jjchester** [comment]
I think once the existing comments get fixed up it'll be good to merge.

File: lib/components/LineChart.dart [hunk 4]
```diff
@@ -0,0 +1,280 @@
+import 'package:fl_chart/fl_chart.dart';
```
Comment (jjchester) [inline lib/components/LineChart.dart]: This whole file can be removed since we never use the line chart

File: lib/components/DoubleBarChart.dart [hunk 5]
```diff
@@ -0,0 +1,374 @@
...
+                                  Padding(
+                                    padding: const EdgeInsets.all(15.0),
+                                    child: DatePicker(),
+                                  )
+                                ],
+                              );
+                            }));
+                          });
+
+                      // refreshState();
```
Comment (jjchester) [inline lib/components/DoubleBarChart.dart]: comment

File: lib/components/WeightChart.dart [hunk 6]
```diff
@@ -0,0 +1,336 @@
...
+                                      ),
+                                    ),
+                                    DatePicker()
+                                  ],
+                                ),
+                              );
+                            }));
+                          });
+
+                      // refreshState();
```
Comment (jjchester) [inline lib/components/WeightChart.dart]: comment

**Comment by jjchester** [comment]
Looks good 👍

**Comment by njoudrey** [comment]
looks great, nice job!

**Pull request merged**